### ISSUE TITLE
Ajouter un thème sombre

### ISSUE THREAD

**Issue by syapk** [zestedesavoir/zds-site#6013]
**Description du besoin**

Ajouter diffèrent thème de couleur et/ou style  pour le site.

En priorité le Black Thème !! ;)

**Description de la solution**

je ne sais pas lol

**Description des alternatives**

...

**Contexte additionnel**

...

**Comment by SpaceFox** [comment]
[Une ressource pour commencer à réfléchir sur ce point](https://zestedesavoir.com/billets/3724/vous-avez-tort-dutiliser-un-mode-sombre-signet/) 😶

**Comment by AmauryCarrade** [comment]
Le thème sombre, c'est aussi (et surtout ?) une question de préférence et de ressenti. Quoiqu'en disent les études (pas mal étant, d'ailleurs, controversés), certaines personnes trouvent un fond sombre plus agréable, et/ou n'ont tout simplement pas la possibilité d'adapter la luminosité de leur environnement et de leur terminal de façon correcte.

Ça ne me semble donc pas être un argument contre l'implémentation d'un thème sombre sur ZdS — sans dire que ça doit se faire, parce que c'est pas forcément trivial, mais cet argument ne suffit pas selon moi.

---

Si thème sombre il y a, je pense que ce n'est pas pour tout de suite. Une piste pourrait être de passer toutes les couleurs du site en variables CSS (très simple à faire depuis #5922), qu'on modifierait si une classe est ajoutée au `body`, mais il faudrait voir si un simple changement des palettes suffirait pour avoir un thème sombre convenable. Peut-être, mais c'est à vérifier — je n'en suis pas sûr.

D'ici là, tu peux utiliser [Dark Reader](https://darkreader.org/) pour avoir un thème sombre automatique sur ZdS. Ça marche bien.

**Comment by SpaceFox** [comment]
Bien sûr, je réagissais surtout au fait que c'est une demande qui arrive avec une "priorité" qui n'est pas si naturelle que ça (même si c'est la mode).
